test/fluree/db/ledger/api/downloaded.clj

```diff
@@ -191,6 +192,36 @@
       ; Are some of the predicates we expect returned?
       (is (every? boolean (map #(predicates %) ["comment/nestedComponent" "person/stringUnique"]))))))
 
+(deftest sign-multi-query
+  (testing "sign-multi-query where collections are not named in alphanumeric order"
+    (let [private-key (slurp "default-private-key.txt")
+          qry-str      "{\"collections\":{\"select\":[\"*\"],\"from\":\"_collection\"},\n       \"predicates\":{\"select\":[\"*\"],\"from\":\"_predicate\"},\n       \"_setting\":{\"select\":[\"*\"],\"from\":\"_setting\"},\n       \"_rule\":{\"select\":[\"*\"],\"from\":\"_rule\"},\n       \"_role\":{\"select\":[\"*\"],\"from\":\"_role\"},\n       \"_user\":{\"select\":[\"*\"],\"from\":\"_user\"}\n      }"
+          request      {:headers {"content-type" "application/json"}
+                        :body    qry-str}
+          q-endpoint   (str endpoint-url "multi-query")
+          signed-req   (http-signatures/sign-request :post q-endpoint request private-key)
+          resp         @(http/post q-endpoint signed-req)
+          responseKeys (keys resp)
+          status       (:status resp)
+          body         (some-> resp :body bs/to-string json/parse)
+          collections  (into #{} (map #(:_collection/name %) (:collections body)))
+          predicates   (into #{} (map #(:_predicate/name %) (:predicates body)))
+          roles        (into #{} (map #(:_role/id %) (:_role body)))]
+
+      (is (= 200 status))
+
+      ; The keys in the response are -> :request-time :aleph/keep-alive? :headers :status :connection-time :body
+      (is (not (empty? (remove nil? (map #(#{:request-time :aleph/keep-alive? :headers :status :connection-time :body} %)
```

Comment:
If you want to test that all of these keys are in responseKeys, you could do `(is (every? (map #(#{...}) responseKeys)))`.

Also if we're not interoperating with an environment that prefers different idioms (e.g. Java / JavaScript), `response-keys` would be a better name.

And are you sure that `:aleph/keep-alive?` could be in there? We're not using aleph anymore.

Reply:
:aleph/keep-alive -> we may need to look at all of the unit tests

Reply:
Made following changes across unit tests:
- renamed responseKeys to response-keys
- when aleph was mentioned, updated response-keys based on http-kit
- above let to a few formatting changes

Reply:
As noted below, we used `text/contains-many?` to check the response keys.